src/Components/Facility/PatientConsultationNotesList.tsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from "react";
+import { RESULTS_PER_PAGE_LIMIT } from "../../Common/constants";
+import CircularProgress from "../Common/components/CircularProgress";
+import useQuery from "../../Utils/request/useQuery";
+import routes from "../../Redux/api";
+import { StateType } from "./models";
+import useSlug from "../../Common/hooks/useSlug";
+import DoctorNote from "./DoctorNote";
+
+interface PatientNotesProps {
+  state: StateType;
+  setState: any;
+  patientId: string;
+  facilityId: string;
+  reload?: boolean;
+  setReload?: any;
+}
+
+const pageSize = RESULTS_PER_PAGE_LIMIT;
+
+const PatientConsultationNotesList = (props: PatientNotesProps) => {
+  const { state, setState, reload, setReload } = props;
+  const consultationId = useSlug("consultation") ?? "";
+
+  const [isLoading, setIsLoading] = useState(true);
+
+  useQuery(routes.getPatientNotes, {
+    pathParams: {
+      patientId: props.patientId,
+    },
+    query: {
+      consultation: consultationId,
+      offset: (state.cPage - 1) * RESULTS_PER_PAGE_LIMIT,
+    },
+    prefetch: reload,
```

Comment:
useQuery itself provides a way to reload. You can do:

```
const { res, data, refetch } = useQuery(...);
```

Reply:
@rithviknishad I need to call the `refetch` method in the parent component as well after a new note is added, I belive it best to have a separate variable for that.